### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by RndIG**

### CODE REVIEW

File: src/main/java/com/carservice/entities/Employee.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.carservice.entities;
+
+
+
+public class Employee {
```
Comment (Alexmansar) [inline src/main/java/com/carservice/entities/Employee.java:5]: why we need it? I mean empty class

File: src/main/java/com/carservice/workshop/Workshop.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import com.carservice.entities.Employee;
+
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class Workshop {
+
+
+
+
```
Comment (Alexmansar) [inline src/main/java/com/carservice/workshop/Workshop.java]: remove this lines please

**Review by Alexmansar — CHANGES_REQUESTED** [review]
@RndIG Hi)) In general good job) few commets)

File: src/main/java/com/carservice/entities/Employee.java (hunk 1 above)
Comment (RndIG) [reply src/main/java/com/carservice/entities/Employee.java:5]: Placeholder for Employee entity, for usage as assigned employee later.
As well as class used as reference to employee currently working on order.

Comment (Alexmansar) [reply src/main/java/com/carservice/entities/Employee.java:5]: Ok) in general, please don't add code that isn't needed yet)

**Pull request merged**